xxhash.h

```diff
@@ -1854,15 +1852,13 @@ XXH32_finalize(xxh_u32 h32, const xxh_u8* ptr, size_t len, XXH_alignment align)
                          /* fallthrough */
            case 4:       XXH_PROCESS4;
                          return XXH32_avalanche(h32);
-
            case 13:      XXH_PROCESS4;
```

Comment:
The extra line is useful for the reader : 
it groups each block into logical `fallthrough` consecutive parts.